packages/babel-parser/test/fixtures/es2015/uncategorised/242/output.json

```diff
@@ -0,0 +1,182 @@
+{
+  "type": "File",
+  "start": 0,
+  "end": 31,
+  "loc": {
+    "start": {
+      "line": 1,
+      "column": 0
+    },
+    "end": {
+      "line": 1,
+      "column": 31
+    }
+  },
+  "errors": [
+    "SyntaxError: Assigning to 'eval' in strict mode (1:15)",
+    "SyntaxError: Binding 'eval' in strict mode (1:15)"
```

Comment:
This error should be deduplicated.

Input code:
```js
"use strict"; (eval = 10) => 42
```

It might be hard to do, because we are first parsing `eval = 10` as an assignment expression, and then converting it to a paremeter later when we find `=>`